File: scratch/update_en_json.js
const fs = require('fs');
const enPath = 'c:/dev/donbrico-net/locales/en.json';
const data = JSON.parse(fs.readFileSync(enPath, 'utf8'));

// Update nav
data.nav.geminiNano = "Gemini Nano";

// Update docs.apiKeys
data.docs = data.docs || {};
data.docs.apiKeys = {
  page: {
    title: "API Keys Guide — Donbrico Autofill AI"
  },
  breadcrumb: {
    home: "Home",
    docs: "Docs",
    title: "API Keys"
  },
  header: {
    badge: "Setup Guide",
    title: "Get your <em>API Keys</em>",
    intro: "Donbrico Autofill AI works best when you connect your own AI provider. Here is how to get your keys from the most popular services.",
    speedNote: {
      title: "Blazing fast.",
      text: "Using your own API key is typically 5-10x faster than the built-in free models and has much higher limits."
    }
  },
  providers: {
    groq: {
      title: "Groq",
      tag: "Fast · Free tier",
      label: "Provider 01",
      url: "console.groq.com",
      pills: {
        free: "Free tier available",
        models: "Llama 3 · Mixtral · Gemma"
      },
      cta: "Create account ↗",
      intro: "Groq runs open-source models on custom silicon (LPUs) — the result is remarkably fast inference. The free tier is generous enough for everyday autofill use at no cost.",
      step1: {
        title: "Sign up for a free Groq account",
        text: "Go to <strong>console.groq.com</strong> and click <strong>Sign up</strong>. Groq supports Google, GitHub, and email signup. No credit card is required for the free tier."
      },
      step2: {
        title: "Create an API key",
        text: "In the left sidebar, click <strong>API Keys</strong>. Click <strong>Create API Key</strong>, enter a name like \"Donbrico\", and click <strong>Submit</strong>.",
        warn: "Copy the key immediately. Groq does not show it again after creation. It begins with <span class=\"code-inline\">gsk_…</span>"
      },
      step3: {
        title: "Paste into the extension",
        text: "Open the extension popup → <strong>Settings → AI Provider → Groq</strong>. Paste your key and save."
      },
      info: "Groq free tier includes rate limits (e.g. 30 requests/minute for Llama 3). For typical autofill usage this is more than enough. If you hit limits, upgrading to a paid plan removes them.",
      table: {
        thModel: "Model",
        thSpeed: "Speed",
        thFree: "Free tier?",
        recommended: "Recommended",
        extremelyFast: "Extremely fast",
        fast: "Fast",
        yes: "Yes",
        yesLower: "Yes (lower limit)"
      }
    },
    openrouter: {
      title: "OpenRouter",
      tag: "100+ models",
      label: "Provider 02",
      url: "openrouter.ai",
      pills: {
        free: "Free models available",
        models: "100+ models from any provider"
      },
      cta: "Create account ↗",
      intro: "OpenRouter is a unified gateway to over 100 AI models — including GPT-4o, Claude, Llama, Mistral, and many more — through a single API key. Several models are permanently free with no credit card required.",
      step1: {
        title: "Create an OpenRouter account",
        text: "Visit <strong>openrouter.ai</strong> and click <strong>Sign In</strong>. OpenRouter supports Google and GitHub sign-in, or you can use an email address."
      },
      step2: {
        title: "Optional: add credits for paid models",
        text: "Go to <strong>Credits</strong> in your account and top up if you want to use paid models like GPT-4o or Claude. For free models only, no payment is needed.",
        tip: "Filter by <strong>Free</strong> on the Models page to see the full list of zero-cost models available right now."
      },
      step3: {
        title: "Create an API key",
        text: "Go to <strong>Keys</strong> in the top navigation. Click <strong>Create Key</strong>, name it \"Donbrico\", and copy the key — it begins with <span class=\"code-inline\">sk-or-…</span>",
        warn: "Copy the key now — it will be hidden after you close this dialog."
      },
      step4: {
        title: "Paste into the extension",
        text: "Open the extension popup → <strong>Settings → AI Provider → OpenRouter</strong>. Paste your key, choose your preferred model, and save."
      },
      table: {
        thModel: "Model",
        thNotes: "Notes",
        thCost: "Cost",
        llamaNote: "Fast, reliable free option",
        mistralNote: "Good for structured outputs",
        gptNote: "Premium quality",
        claudeNote: "Premium quality",
        free: "Free",
        payAsYouGo: "Pay-as-you-go"
      }
    },
    anthropic: {
      title: "Anthropic",
      tag: "Claude",
      label: "Provider 03",
      url: "console.anthropic.com",
      pills: {
        payAsYouGo: "Pay-as-you-go",
        models: "Claude 3.5 Haiku · Claude 3.5 Sonnet"
      },
      cta: "Create account ↗",
      intro: "Anthropic's Claude models are known for following instructions precisely — ideal for generating well-structured cover letters and professional answers.",
      step1: {
        title: "Create an Anthropic account",
        text: "Visit <strong>console.anthropic.com</strong> and click <strong>Sign up</strong>. You can use Google or an email address."
      },
      step2: {
        title: "Add billing information",
        text: "From the Console dashboard, go to <strong>Settings → Billing</strong>. Click <strong>Add credit card</strong> and complete the form. Anthropic is pay-as-you-go with no monthly fee.",
        tip: "You can set a monthly spend limit under Billing → Usage limits to avoid any surprise charges."
      },
      step3: {
        title: "Generate an API key",
        text: "In the left sidebar, click <strong>API Keys</strong>. Click <strong>Create Key</strong>, name it \"Donbrico\", and copy the key — it starts with <span class=\"code-inline\">sk-ant-…</span>",
        warn: "The key is shown only once at creation time. Copy it before closing the dialog."
      },
      step4: {
        title: "Paste into the extension",
        text: "Open the extension popup → <strong>Settings → AI Provider → Anthropic</strong>. Paste your key and save."
      },
      table: {
        thModel: "Model",
        thSpeed: "Speed",
        thCost: "Cost (approx. per autofill)",
        veryFast: "Very fast",
        moderate: "Moderate",
        costHaiku: "~$0.001",
        costSonnet: "~$0.006"
      }
    },
    openai: {
      title: "OpenAI",
      tag: "GPT-4o",
      label: "Provider 04",
      url: "platform.openai.com",
      pills: {
        payAsYouGo: "Pay-as-you-go",
        models: "GPT-4o · GPT-4o mini"
      },
      cta: "Create account ↗",
      intro: "OpenAI is the most widely used provider. GPT-4o mini is fast, accurate, and costs fractions of a cent per autofill.",
      step1: {
        title: "Create a free account",
        text: "Go to <strong>platform.openai.com</strong> and sign up. You can use Google, Microsoft, or email."
      },
      step2: {
        title: "Add a payment method",
        text: "Navigate to <strong>Settings → Billing → Add payment method</strong>. OpenAI requires a credit card before generating API keys. You will only be charged for actual usage — new accounts receive $5 in free credits."
      },
      step3: {
        title: "Create an API key",
        text: "Go to <strong>API Keys</strong> in the left sidebar (or visit <span class=\"code-inline\">platform.openai.com/api-keys</span>). Click <strong>Create new secret key</strong>, give it a name like \"Donbrico\", and click <strong>Create secret key</strong>.",
        warn: "Copy your key immediately — OpenAI only shows it once. Store it somewhere safe."
      },
      step4: {
        title: "Paste the key into the extension",
        text: "Open the Donbrico Autofill AI popup → <strong>Settings → AI Provider → OpenAI</strong>. Paste your key (it starts with <span class=\"code-inline\">sk-…</span>) and click Save."
      },
      table: {
        thModel: "Model",
        thSpeed: "Speed",
        thCost: "Cost (approx. per autofill)",
        costMini: "~$0.001",
        costFull: "~$0.005"
      }
    }
  },
  comparison: {
    title: "Quick comparison",
    thProvider: "Provider",
    thFreeTier: "Free tier?",
    thCardRequired: "Card required?",
    thBestFor: "Best for",
    thKeyPrefix: "Key prefix",
    groqBest: "Fastest responses, no cost",
    openRouterBest: "Flexibility, free models",
    anthropicBest: "Precise instruction-following",
    openaiBest: "Best overall quality",
    signupCredit: "$5 credit on signup",
    no: "No",
    yes: "Yes",
    yesFreeModels: "No (free models)"
  },
  proTip: {
    label: "Pro tip",
    text: "No API key needed at all — if your computer supports it, <strong>Gemini Nano</strong> runs directly in Chrome for free with complete privacy.",
    link: "Check Gemini Nano compatibility →"
  }
};

fs.writeFileSync(enPath, JSON.stringify(data, null, 2) + '\n');
console.log('en.json updated successfully.');

File: scratch/update_en_json.py
import json
import os

file_path = r'c:\dev\donbrico-net\locales\en.json'

with open(file_path, 'r', encoding='utf-8') as f:
    data = json.load(f)

# Add nav item
if 'nav' not in data:
    data['nav'] = {}
data['nav']['extensionsRecruitReply'] = "RecruitReply AI"

# Add recruitReply section
data['recruitReply'] = {
  "page": {
    "title": "RecruitReply AI – Reply to Candidates in Seconds (Free Chrome Extension)",
    "description": "Reply to candidates on LinkedIn and ATS in seconds. RecruitReply AI drafts professional messages inside your inbox. Free Chrome extension.",
    "ogTitle": "RecruitReply AI – Reply to Candidates in Seconds (Free Chrome Extension)",
    "ogDescription": "Reply to candidates on LinkedIn and ATS in seconds. RecruitReply AI drafts professional messages inside your inbox. Free Chrome extension."
  },
  "hero": {
    "eyebrow": "CHROME EXTENSION",
    "title": "Reply to candidates in seconds.",
    "subtitle": "RecruitReply AI drafts professional messages directly inside your LinkedIn or ATS inbox, for free. Includes template library for sourcing, follow-ups, and offers.",
    "buttonPrimary": "Add to Chrome — Free",
    "buttonSecondary": "See how it works ↓",
    "disclaimer": "🔒 Built-in AI runs locally — data stays in your browser"
  },
  "howItWorks": {
    "sectionTitle": "How It Works",
    "step1": {
      "title": "1. Save your templates",
      "description": "Store your sourcing outreach, follow-ups, offers, and rejection letters in the library."
    },
    "step2": {
      "title": "2. Open your ATS or LinkedIn",
      "description": "Works directly inside LinkedIn, Gmail, Greenhouse, Lever, and Workday."
    },
    "step3": {
      "title": "3. AI drafts the reply",
      "description": "RecruitReply uses your template, reads the context, and drafts a personalized response instantly."
    }
  },
  "features": {
    "sectionTitle": "Features",
    "feature1": {
      "title": "Works where you work",
      "description": "Handles LinkedIn InMails and ATS email composers automatically."
    },
    "feature2": {
      "title": "Smart Variables",
      "description": "Save your templates with {{name}}, {{role}}, and {{company}}. AI replaces them dynamically."
    },
    "feature3": {
      "title": "Privacy First",
      "description": "Your templates are stored locally. Use on-device Chrome AI to keep everything completely private."
    },
    "feature4": {
      "title": "Bring Your Own AI",
      "description": "Connect OpenAI, Claude, or Gemini using your own API key if you prefer."
    }
  },
  "pricing": {
    "sectionTitle": "Simple pricing. No hidden costs.",
    "free": {
      "title": "Free",
      "price": "$0",
      "features": {
        "f1": "5 Saved Reply templates",
        "f2": "1 ATS profile",
        "f3": "5 core smart variables",
        "f4": "10 AI replies/day",
        "f5": "3 Reply Scenarios",
        "f6": "Single BYOK provider",
        "f7": "3 recruiter tones"
      },
      "button": "Add to Chrome"
    },
    "proMonthly": {
      "title": "Pro (Monthly)",
      "price": "$7.99/mo",
      "features": {
        "f1": "Unlimited AI replies",
        "f2": "Unlimited saved replies",
        "f3": "Unlimited profiles",
        "f4": "5 recruiter tones",
        "f5": "Export/import templates",
        "f6": "20+ Smart variables",
        "f7": "Multiple BYOK providers",
        "f8": "Priority support"
      },
      "button": "Subscribe monthly"
    },
    "proLifetime": {
      "title": "Pro (Lifetime)",
      "price": "$69",
      "tagline": "Pay once, use forever",
      "features": {
        "f1": "Everything in Pro Monthly",
        "f2": "No recurring fees",
        "f3": "Future updates included",
        "f4": "Early adopter pricing"
      },
      "button": "Lifetime pass"
    }
  },
  "faq": {
    "sectionTitle": "Frequently Asked Questions",
    "q1": {
      "question": "Does it work with LinkedIn and major ATS platforms?",
      "answer": "Yes, it works directly inside LinkedIn InMail, Gmail, and most ATS platforms like Greenhouse and Lever."
    },
    "q2": {
      "question": "Is it really free?",
      "answer": "Yes! The free plan gives you everything you need to start, including 5 saved templates for your library."
    },
    "q3": {
      "question": "Are candidates' data safe?",
      "answer": "Yes, all processing stays on-device with Gemini Nano (Chrome). If you use your own API key, data goes only to that provider. Donbrico never sees your messages."
    }
  }
}

with open(file_path, 'w', encoding='utf-8') as f:
    json.dump(data, f, indent=2, ensure_ascii=False)

print("Updated en.json with recruitReply keys.")
